openedx/core/djangoapps/course_roles/readme.rst

```diff
@@ -0,0 +1,15 @@
+Course Roles
```

Comment:
There are two course_roles/readme.rst files (one uppercase README and the other lowercase). Please remove the duplicate so there is only one.